README.md

```diff
@@ -40,6 +40,14 @@ To run the test-suite locally, first complete the bootstrap step as outline abov
 tox -e tests
 ```
 
+### Documentation
+
+To generate html documentation locally in docs/_build/html, run the following command
+```shell 
+# Generate doc with tox
+tox -e docs
+```
+
```

Comment:
We could also add a friendlier command that allows you to serve built docs with a simple fileserver:

```
# Watch and server HTML docs
tox -e docs -- livehtml
```

In the future, would be nice to have a set of `doit` targets to build/watch docs in various formats (HTML/PDF), relying on the Dockerfile we have available.

Reply:
watch and **serve** ?

Reply:
> In the future, would be nice to have a set of doit targets to build/watch docs in various formats (HTML/PDF), relying on the Dockerfile we have available.

+1
We should create an issue for that (otherwise we may forget)

Reply:
we can open the doc from the browser (by targeting the index.html), what is the value to run it in webserver ? ...